troubleshooting/known_issues/enable_iscsid_multipathd.md

```diff
@@ -0,0 +1,214 @@
+...
+```
+
+## Workaround Description
+
+Step-1: Get the version of the “csm-packages” from compute node BOS session template
+
+**Example**
+
+```bash
+ncn-m001:~ # cray bos sessiontemplates  list | grep compute-*
+…
+enable_cfs = true
+name = "compute-25.1.0-alpha2.x86_64-csm-160-rc4”
+…
+```
+
+```bash
+ncn-m001:~ # cray bos sessiontemplates describe compute-25.1.0-alpha2.x86_64-csm-160-rc4  --format json
+{…
+  "cfs": {
+    "configuration": "compute-25.1.0-alpha2-csm-160-rc4"
+  },
+…
+}
+```
+
+```bash
+ncn-m001:~ # cray cfs configurations describe compute-25.1.0-alpha2-csm-160-rc4 --format json
+{
+  "lastUpdated": "2024-11-02T12:15:21Z",
+  "layers": [
+    {
+      "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+      "commit": "d530f0a277c9d5dc9e3cb487d32d6b316757f00e",
+      "name": "csm-packages-1.6.0-rc.4", 
+      "playbook": "csm_packages.yml"
+    }, 
+…
+```
+
+Step-2: Get the corresponding csm-config branch (@VCS) from product catalog given “csm-packages-*” version found from Step-1
+
+**Example**
+
+```bash
+ncn-m001:~ # kubectl get cm -n services cray-product-catalog -o yaml | yq - r 'data.csm' | grep ^1.6.0-rc.4: -A 10
+1.6.0-rc.4:
+  configuration:
+    clone_url: https://vcs.cmn.fanta.hpc.amslabs.hpecorp.net/vcs/cray/csm-config-management.git
+    commit: d530f0a277c9d5dc9e3cb487d32d6b316757f00e
+    import_branch: cray/csm/1.27.2
+```
+
+Step-3: Log into VCS and clone “csm-config-management.git” @ VCS
+
+```bash
+ncn-m001:/home/ # USERNAME=$( kubectl get secrets -n services vcs-user-credentials -o json | jq -r .data.vcs_username | base64 -d  )
+ncn-m001:/home/ # PSWD=$( kubectl get secrets -n services vcs-user-credentials -o json | jq -r .data.vcs_password | base64 -d  )
+ncn-m001:/home/ # git clone https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+```
+
+**Note:** use above $USERNAME and $PSWD for VCS login
+
+Step-4: Apply fix against import_branch found in Step-2
+
+**Example**
+
+```bash
+ncn-m001:/home/ # cd csm-config-management
+ncn-m001:/home/csm-config-management # git checkout cray/csm/1.27.2
+ncn-m001:/home/csm-config-management # git checkout -b CASMTRIAGE-7509
+```
+**Note: ** `cray/csm/1.27.2` is a target branch and `CASMTRIAGE-7509` is a new branch
+
+Add new role to enable `iscsid` and `multipathd` service:
+
+```bash
+ncn-m001:~/csm-config-management # cat roles/csm.enable_iscsid_multipathd/tasks/main.yml
+---
+- name: Ensure iscsid service is started
+  ansible.builtin.systemd:
+    name: iscsid
+    state: started
+    enabled: true
+
+- name: Ensure multipathd service is started
+  ansible.builtin.systemd:
+        name: multipathd
+        state: started
+        enabled: true
+ncn-m001:~/csm-config-management #
+```
+
+Apply below changes to “csm-config-management/csm_packages.yml” to Application-nodes only play and Compute-nodes only play under “csm_services” in order to enable `iscsid` and `multipathd` services.
```

Comment:
Change to: "Apply **_the following_** changes"
Also use back-tics, not double quotes, for `csm-config-management/csm_packages.yml`